problem-27/README.md

```diff
@@ -0,0 +1,15 @@
+# Task 27: Apply Can deactivate guard to admin route (lv 3)
+
+## Description
+Update the 'canDeactivateGuard' to stop navigation away from the 'AdminComponent'. The guard should call the 'getIsUserAdmin()' function
```

Comment:
This doesn't make sense. Does it mean that a user without an admin role can never leave the AdminComponent component?

In the real world, we use canActivate to check whether a user has completed the task they started, for example, filling out a form.

Reply:
Resolved - replaced with another question.